hg-fast-export.py

```diff
@@ -447,6 +452,8 @@ def bail(parser,opt):
       help="Read branch map from BRANCHESFILE")
   parser.add_option("-T","--tags",dest="tagsfile",
       help="Read tags map from TAGSFILE")
+  parser.add_option("-c",action="store_true",dest="commitmsg",
```

Comment:
Is '-c' the right flag, likewise, the name 'commitmsg' does not match the option.